src/content/docs/infrastructure/host-integrations/host-integrations-list/postfix-monitoring-integration.mdx

```diff
@@ -0,0 +1,406 @@
+            - run: cat /var/log/mail.log | pflogsumm | grep -C 11 'Senders by message count' -B 1 -A 9 | sed '/Recipients by message count/q'
+              split: horizontal
+              header_split_by: (.*.)
+              set_header: [sender.msgCount, senders]
+              regex_match: true
+              split_by: \s+(\d+)\s+(.*)
+            - run: cat /var/log/mail.log | pflogsumm | grep -C 11 'Recipients by message count' -B 1 -A 9 | sed '/Senders by message size/q'
+              split: horizontal
+              header_split_by: (.*.)
+              set_header: [recipient.msgCount, recipients]
+              regex_match: true
+              split_by: \s+(\d+)\s+(.*)   
+
+    #Read the daemon message details.                                       
+        - event_type: PostfixDaemonMessage
+          commands:    
+            - run: cat /var/log/mail.log | pflogsumm | grep -A 10 'Master daemon messages'
+              split: horizontal
+              set_header: [daemon.msgCount, daemonMessage]
+              regex_match: true
+              split_by: \s*(\d+)\s+(\w+.*.)
+              
+    #Read the message deferral details.  
+        - event_type: PostfixDeferralMessage
+          commands:    
+            - run: cat /var/log/mail.log | pflogsumm | grep -A 15 'message deferral detail' | sed '/message bounce detail (by relay)/q'
+              split: horizontal
+              header_split_by: (\s+\w+\s+.*)
+              set_header: [deferral.msgCount, deferralMessage]
+              regex_match: true
+              split_by: \s*(\d+)\s+(\w+.*.)
+
+    #Read the warning message details.            
+        - event_type: PostfixWarningMessage
+          commands:    
+            - run: cat /var/log/mail.log | pflogsumm | grep -v 'mail_queue_enter' | grep -C 100 'Warnings' -B 1 -A 10 | sed '/Fatal Errors/q'
+              split: horizontal
+              header_split_by: \s+(\w+)\s+(.*)
+              set_header: [Warning.msgCount, warningMessage]
+              regex_match: true
+              split_by: \s*(\d+)\s\s\s(.*.)
+
+    #Read the error message details.  
+        - event_type: PostfixErrorMessage
+          commands:    
+            - run: cat /var/log/mail.log | pflogsumm | grep -C 15 'Fatal Errors' -B 1 -A 10 | sed '/Panics/q'
+              split: horizontal
+              header_split_by: \s+(\w+)\s+(.*)
+              set_header: [Fatal.msgCount, fatalMessage]
+              regex_match: true
+              split_by: \s*(\d+)\s\s\s(.*.)
+
+    #Read the message bounce details.              
+        - event_type: PostfixBounceMessage
+          commands:    
+            - run: cat /var/log/mail.log | pflogsumm | grep -A 10 'message bounce detail (by relay)' | sed '/message reject detail/q'
+              split: horizontal
+              header_split_by: \s+(\w+)\s+(.*)
+              set_header: [Bounce.msgCount, bounceMessage]
+              regex_match: true
+              split_by: \s*(\d+)\s\s\s(.*.)
```

Comment:
Indentation was perfect until line 140.

Need to align the indentation of lines 143 to 325 with line 134

Reply:
done